src/quo/components/share/share_qr_code/style.cljs

```diff
@@ -1,45 +1,120 @@
+(def header-tab-inactive {:background-color colors/white-opa-5})
+(def space-between-tabs {:width 8})
+
+(def info-icon
+  {:margin-left :auto
+   :align-self  :center})
+
+(def info-icon-color colors/white-opa-40)
+
+;; QR code
+(defn qr-code-size
+  [total-width]
+  (- total-width (* 2 padding-horizontal)))
+
+(defn qr-image-background-color
+  [customization-color]
+  (colors/resolve-color customization-color :light))
+
+;; Bottom part
+(def bottom-container
+  {:margin-top      8
+   :flex-direction  :row
+   :justify-content :space-between})
 
-(def profile-address-content-container
-  {:padding-top 2
-   :align-self  :flex-start})
+(def title {:color colors/white-opa-40})
 
-(def profile-address-content
-  {:color colors/white})
+(def share-button-size 32)
+(def ^:private share-button-gap 16)
 
-(def profile-address-label
-  {:color colors/white-opa-40})
+(defn share-button-container
+  [alignment]
+  {:justify-content (if (= alignment :top) :flex-start :center)
+   :margin-left     share-button-gap})
 
-(def share-button-container
-  {:flex            1
-   :flex-direction  :column
-   :justify-content :center
-   :align-items     :flex-end})
+(defn data-text
+  [total-width]
+  {:width (- total-width (* 2 padding-horizontal) share-button-size share-button-gap)})
 
-(def icon-container
-  {:height          36
+;; Wallet variants
+(def wallet-data-and-share-container
+  {:margin-top      2
    :flex-direction  :row
-   :align-items     :center
+   :justify-content :space-between})
+
+(def wallet-legacy-container {:flex 1})
+
+(def wallet-multichain-container {:flex 1 :margin-top 4})
+
+(def wallet-multichain-networks
+  {:flex-direction  :row
    :justify-content :space-between
-   :padding-bottom  12})
+   :margin-bottom   8})
+
+(def wallet-multichain-data-container {:margin-top 4})
+
+;; Dashed line
+(def divider-container
+  {:height            8
+   :margin-horizontal 4
+   :justify-content   :center
+   :overflow          :hidden})
+
+(def ^:private padding-for-divider (+ padding-horizontal 4))
+(def ^:private dashed-line-width 2)
+(def ^:private dashed-line-space 4)
+
+(def dashed-line
+  {:flex-direction :row
+   :margin-left    -1})
+
+(def line
+  {:background-color colors/white-opa-20
+   :width            dashed-line-width
+   :height           1})
+
+(def line-space
+  {:width  dashed-line-space
+   :height 1})
+
+(defn number-lines-and-spaces-to-fill
+  [component-width]
+  (let [line-and-space-width (+ dashed-line-width dashed-line-space)
+        width-to-fill        (- component-width (* 2 padding-for-divider))
+        number-of-lines      (* (/ width-to-fill line-and-space-width) 2)]
+    (inc (int number-of-lines))))
+
+(def ^:private get-network-full-name
+  {"eth"  :ethereum
+   "opt"  :optimism
+   "arb1" :arbitrum})
+
+(defn network-short-name-text
+  [network-short-name]
+  {:color (-> network-short-name
+              (get-network-full-name :unknown)
+              (colors/resolve-color nil))})
```

Comment:
Why resolve nil? Better to pass theme if that is what's happening here

Reply:
This component doesn't have theme variants.

`(resolver-color color nil)` is being used because network colors don't have a dark/light variant and since this component is not "theme sensitive", then we don't have the `theme` prop available

Reply:
Got it, I think @ilmotta was suggesting we pass them so this is future proofed should we need to adjust the Colors the colors mechanism.

Reply:
I think if they add a dark color for the networks, they will also update figma to show light/blur variants of those components, IMO, right now we'd be creating code speculating. I'm not sure if it's worth it :thinking: 

If you think we should do it, then I can change it :slightly_smiling_face: